### DISCUSSION TITLE
Adding Integration

### DISCUSSION THREAD

**Discussion by mcfarlde** [Tasshack/dreame-vacuum#497] [General]
I added v1.0.3 via HACS and when adding the integration, get...Config flow could not be loaded: Unknown error. Any ideas?
UPDATE:  Setting logging level higher got me to the Xaomi Miio login page.  Using the credentials Account ID: with a freshly reset password failed, using the third party Login in the Dream appal Apple ID also failed.

**Comment by Tasshack** [comment]
You need to install the Beta version of the integration to be able to login using Dreame home account.

**Reply by mcfarlde** [reply]
Thanks, I will give that a try.  I removed the integration using HACS and now it doesn't show to add again.  I'll try the link in Github.

**Comment by mcfarlde** [comment]
I got the map part working but strangely can't get the vacuum card working.  I see :Custom element doesn't exist: xiaomi-vacuum-card even though I have it in Manage Resources as a javascript module /hacsfiles/lovelace-xiaomi-vacuum-card/xiaomi-vacuum-card.js and the file exist in the www folder.  Is the issue the file location being /hacsfiles/lovelace in Reources when I put it in www folder?

**Comment by Tasshack** [comment]
Everything you just wrote is related to the map card which is another project not related to dreame vacuum. This is a backend project it has nothing to do with the lovelace.

**Comment by mcfarlde** [comment]
Thanks, got it working after adding to configuration.yaml.
